packages/auth/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@standardnotes/auth",
-  "version": "3.2.0",
```

Comment:
This would be a breaking change for the dependent `auth` repo. I think we should change the MINOR version at least?

Reply:
good point